### PR TITLE
Bic 5210 response code and country 

### PR DESCRIPTION
**Pull request by ryanlijiang**

### CODE REVIEW

File: source/docs/references/batch_payment/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The following parameters are passed as a JSON object on a Content-Disposition he
 
 ### Format of data in file
 
-The API expects a single CSV file with one transaction per row. It does not expect a header row.
+The API expects a single CSV file with one transaction per row. It does not expect a header row and does not allow commas, unless the name and address are in a payment profile.
```
Comment (shawn9001) [inline source/docs/references/batch_payment/index.md]: This makes is sound like if the name and address are in the file then commas are fine in the file.

Maybe try this?

The API expects a single CSV file with one transaction per row.  It does not expect a header row.  None of the fields in the file may contain a comma including the address fields.  If the transaction is referencing a Payment Profile through populating the Customer Code field in the uploading file, commas in the profile name and address are supported.

File: source/docs/references/batch_payment/index.md [hunk 2]
```diff
@@ -59,9 +59,7 @@ For batches of EFT transactions, the API expects the following columns:
 - **Province** - Province ID for the recipient. This field is optional, with a maximum length of 2 letters.
     - If the country ID is CA or US, the province ID must be populated with a valid province ID for the respective country.
     - If the country ID is provided and not set to a value of 'CA' or 'US' then Province Id must be empty.
-- **Country** - Country ID for the recipient. This field is optional, with a maximum length of 2 letters.
-    -  CA - Canada
-    -  US - United States
+- **Country** - Country ID for the recipient. This field is optional and accepts two-letter ISO country codes. For guidance, please refer to https://en.wikipedia.org/wiki/ISO_3166-2.
 - **Postal Code** - Postal Code for the recipient. This field is optional, with a maximum length of 16 characters.
```
Comment (ryanlijiang) [inline source/docs/references/batch_payment/index.md:63]: test

File: source/docs/references/batch_payment/index.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ The following parameters are passed as a JSON object on a Content-Disposition he
 
 ### Format of data in file
 
-The API expects a single CSV file with one transaction per row. It does not expect a header row.
+The API expects a single CSV file with one transaction per row. It does not expect a header row and does not allow commas, unless the name and address are in a payment profile.
```
Comment (ryanlijiang) [inline source/docs/references/batch_payment/index.md:36]: test

File: source/docs/references/batch_payment/index.md (hunk 2 above)
Comment (ryanlijiang) [inline source/docs/references/batch_payment/index.md:63]: test

**Review by shawn9001 — APPROVED** [review]

**Pull request merged**